integration-tests/steps/change_sets.py

```diff
@@ -1,4 +1,4 @@
-from behave import *
```

Comment:
This is poorly worked in their documentation, you only use what you call, which is given, when, then.

Their documentation,  has examples which do not import *

If you want i can update to match behave documentation instead of best practice

Reply:
Could you update this to match Sean's suggestion?